From dafa1c8ce652b0c3bbac5fc7d294e09900e21040 Mon Sep 17 00:00:00 2001
Message-Id: <dafa1c8ce652b0c3bbac5fc7d294e09900e21040@dist-git>
From: Andrea Bolognani <abologna@redhat.com>
Date: Wed, 5 Aug 2015 18:18:20 +0200
Subject: [PATCH] nodeinfo: Add nodeGetPresentCPUBitmap() to
 libvirt_private.syms

(cherry picked from commit c30ae1864f7b542c3bf765f9b5988f5298dadc9b)

Bug: https://bugzilla.redhat.com/show_bug.cgi?id=1213713

Signed-off-by: Andrea Bolognani <abologna@redhat.com>
Signed-off-by: Jiri Denemark <jdenemar@redhat.com>
---
 src/libvirt_private.syms | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/libvirt_private.syms b/src/libvirt_private.syms
index 1a03268..e92d2d0 100644
--- a/src/libvirt_private.syms
+++ b/src/libvirt_private.syms
@@ -1007,6 +1007,7 @@ nodeGetInfo;
 nodeGetMemory;
 nodeGetMemoryParameters;
 nodeGetMemoryStats;
+nodeGetPresentCPUBitmap;
 nodeSetMemoryParameters;
 
 
-- 
2.5.0

